geode-for-redis/src/main/java/org/apache/geode/redis/internal/services/PassiveExpirationManager.java

```diff
@@ -76,8 +77,8 @@ private void doDataExpiration(RegionProvider regionProvider) {
       }
     } catch (CacheClosedException ignore) {
     } catch (RuntimeException | Error ex) {
-      logger.warn("Passive expiration failed. Will try again in 1 second.",
-          ex);
+      logger.warn("Passive expiration failed. Will try again in " + passiveExpirationInterval
```

Comment:
Something you don't need to clean up in this pr but that you remind me of is that I got the naming of this expiration manager wrong. Native Redis calls this "active" expiration. The expiration we do when we do an op and find the key expired is what native redis calls "passive" expiration. So these stats and log messages may be a bit confusing to native redis users.

Reply:
I've changed all mentions of passive expiration to active expiration.

Reply:
Thank you!